src/main/java/suneido/runtime/builtin/Lucene.java

```diff
@@ -59,15 +61,19 @@ protected Object newInstance(Object... args) {
 
 	@Params("dir")
 	public static Object AvailableQ(Object self, Object a) {
-		String path = Ops.toStr(a);
-		try (FSDirectory dir = FSDirectory.open(new File(path));
+		try	(FSDirectory dir = FSDirectory.open(getPath(a));
 				DirectoryReader reader = DirectoryReader.open(dir)) {
 			new IndexSearcher(reader);
 			return true;
 		} catch (IOException e) {
 			return false;
 		}
 	}
+	
+	@Params("dir")
```

Comment:
@Params is only needed for methods that are callable from Suneido code.

Reply:
Ah, I will clean that up then